voice: stopped
voice: started
2025-07-24 00:00:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 00:05:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 00:10:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 00:15:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 00:20:02 Voice Service not healthy. Restarting...
voice: stopped
2025-07-24 00:25:01 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
2025-07-24 00:30:02 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
voice: started
2025-07-24 00:35:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 00:40:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 00:45:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 00:50:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 00:55:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 01:00:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 01:05:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 01:10:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 01:15:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 01:20:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 01:25:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 01:30:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 01:35:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 01:40:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 01:45:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 01:50:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 01:55:02 Voice Service not healthy. Restarting...
voice: stopped
2025-07-24 02:00:02 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
voice: started
2025-07-24 02:05:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 02:10:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 02:15:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 02:20:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 02:25:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 02:30:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 02:35:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 02:40:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 02:45:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 02:50:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 02:55:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 03:00:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 03:05:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 03:10:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 03:15:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 03:20:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 03:25:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 03:30:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 03:35:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 03:40:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 03:45:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 03:50:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 03:55:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 04:00:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 04:05:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 04:10:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 04:15:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 04:20:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 04:25:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 04:30:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 04:35:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 04:40:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 04:45:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 04:50:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 04:55:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 05:00:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 05:05:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 05:10:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 05:15:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 05:20:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 05:25:02 Voice Service not healthy. Restarting...
voice: stopped
2025-07-24 05:30:02 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
voice: started
2025-07-24 05:35:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 05:40:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 05:45:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 05:50:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 05:55:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 06:00:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 06:05:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 06:10:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 06:15:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 06:20:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 06:25:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 06:30:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 06:35:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 06:40:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 06:45:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 06:50:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 06:55:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 07:00:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 07:05:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 07:10:02 Voice Service not healthy. Restarting...
voice: stopped
2025-07-24 07:15:01 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
voice: started
2025-07-24 07:20:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 07:25:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 07:30:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 07:35:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 07:40:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 07:45:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 07:50:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 07:55:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 08:00:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 08:05:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 08:10:02 Voice Service not healthy. Restarting...
voice: stopped
2025-07-24 08:15:02 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
voice: started
2025-07-24 08:20:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 08:25:06 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 08:30:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 08:35:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 08:40:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 08:45:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 08:50:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 08:55:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 09:00:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 09:05:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 09:10:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 09:15:02 Voice Service not healthy. Restarting...
voice: stopped
2025-07-24 09:20:02 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
voice: started
2025-07-24 09:25:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 09:30:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 09:35:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 09:40:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 09:45:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 09:50:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 09:55:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 10:00:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 10:05:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 10:10:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 10:15:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 10:20:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 10:25:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 10:30:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 10:35:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 10:40:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 10:45:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 10:50:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 10:55:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 11:00:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 11:05:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 11:10:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 11:15:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 11:20:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 11:25:02 Voice Service not healthy. Restarting...
voice: stopped
2025-07-24 11:30:02 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
2025-07-24 11:35:01 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
voice: started
2025-07-24 11:40:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 11:45:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 11:50:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 11:55:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 12:00:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 12:05:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 12:10:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 12:15:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 12:20:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 12:25:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 12:30:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 12:35:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 12:40:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 12:45:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 12:50:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 12:55:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 13:00:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 13:05:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 13:10:02 Voice Service not healthy. Restarting...
voice: stopped
2025-07-24 13:15:02 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
voice: started
2025-07-24 13:20:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 13:25:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 13:30:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 13:35:02 Voice Service not healthy. Restarting...
voice: stopped
2025-07-24 13:40:01 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
voice: started
2025-07-24 13:45:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 13:50:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 13:55:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 14:00:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 14:05:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 14:10:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 14:15:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 14:20:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 14:25:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 14:30:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 14:35:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 14:40:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 14:45:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 14:50:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 14:55:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 15:00:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 15:05:02 Voice Service not healthy. Restarting...
voice: stopped
2025-07-24 15:10:02 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
voice: started
2025-07-24 15:15:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 15:20:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 15:25:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 15:30:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 15:35:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 15:40:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 15:45:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 15:50:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 15:55:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 16:00:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 16:05:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 16:10:01 Voice Service not healthy. Restarting...
voice: stopped
2025-07-24 16:15:02 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
voice: started
2025-07-24 16:20:02 Voice Service not healthy. Restarting...
voice: stopped
2025-07-24 16:25:01 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
voice: started
2025-07-24 16:30:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 16:35:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 16:40:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 16:45:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 16:50:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 16:55:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 17:00:01 Voice Service not healthy. Restarting...
voice: stopped
2025-07-24 17:05:02 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
voice: started
2025-07-24 17:10:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 17:15:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 17:20:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 17:25:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 17:30:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 17:35:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 17:40:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 17:45:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 17:50:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 17:55:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 18:00:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 18:05:01 Voice Service not healthy. Restarting...
voice: stopped
2025-07-24 18:10:02 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
voice: started
2025-07-24 18:15:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 18:20:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 18:25:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 18:30:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 18:35:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 18:40:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 18:45:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 18:50:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 18:55:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 19:00:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 19:05:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 19:10:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 19:15:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 19:20:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 19:25:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 19:30:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 19:35:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 19:40:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 19:45:01 Voice Service not healthy. Restarting...
voice: stopped
2025-07-24 19:50:01 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
voice: started
2025-07-24 19:55:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 20:00:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 20:05:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 20:10:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 20:15:02 Voice Service not healthy. Restarting...
voice: stopped
2025-07-24 20:20:01 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
voice: started
2025-07-24 20:25:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 20:30:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 20:35:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 20:40:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 20:45:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 20:50:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 20:55:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 21:00:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 21:05:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 21:10:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 21:15:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 21:20:01 Voice Service not healthy. Restarting...
voice: stopped
2025-07-24 21:25:02 Voice Service not healthy. Restarting...
Traceback (most recent call last):
  File "/home/voice-recognition/check_service.py", line 62, in <module>
    main()
  File "/home/voice-recognition/check_service.py", line 53, in main
    kill_matching_processes(target_cmd)
  File "/home/voice-recognition/check_service.py", line 37, in kill_matching_processes
    cmdline = ' '.join(proc.info['cmdline'])
TypeError: can only join an iterable
voice: ERROR (not running)
voice: started
2025-07-24 21:30:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 21:35:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 21:40:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 21:45:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 21:50:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 21:55:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 22:00:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 22:05:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 22:10:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 22:15:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 22:20:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 22:25:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 22:30:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 22:35:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 22:40:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 22:45:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 22:50:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 22:55:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 23:00:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 23:05:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 23:10:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 23:15:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 23:20:01 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 23:25:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 23:30:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 23:35:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 23:40:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 23:45:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 23:50:02 Voice Service not healthy. Restarting...
voice: stopped
voice: started
2025-07-24 23:55:02 Voice Service not healthy. Restarting...
